models/iris/iris.sdf.jinja

```diff
@@ -344,6 +329,22 @@
       <gravity>1</gravity>
       <velocity_decay/>
     </link>
+    <joint name='rotor_3_joint' type='revolute'>
```

Comment:
What's this for rotor3 about? Was this missing?

Reply:
I think it just switched the postiion and the git diff got confused